lib/command/definitions.js

```diff
@@ -2,160 +2,65 @@ const getConfig = require('./utils').getConfig;
-  at(position: number): Locator;
-  first(): Locator;
-  last(): Locator;
-  inside(locator: LocatorOrString): Locator;
-  before(locator: LocatorOrString): Locator;
-  after(locator: LocatorOrString): Locator;
-  withText(text: string): Locator;
-  withAttr(attrs: object): Locator;
-  as(output: string): Locator;
+declare namespace CodeceptJS {
+  export type I = Translate<methods, translation["actions"]> & methods;
 }
 
+type ICodeceptCallback = (i: CodeceptJS.I${callbackParams}) => void;
 
-declare function actor(customSteps?: {
-  [action: string]: (this: CodeceptJS.I, ...args: any[]) => void
-}): CodeceptJS.{{I}};
-declare function actor(customSteps?: {}): CodeceptJS.{{I}};
-declare function Feature(title: string, opts?: {}): FeatureConfig;
-declare const Scenario: {
-  (title: string, callback: ICodeceptCallback): ScenarioConfig;
-  (title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-  only(title: string, callback: ICodeceptCallback): ScenarioConfig;
-  only(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-}
 declare interface IScenario {
-  Scenario(title: string, callback: ICodeceptCallback): ScenarioConfig;
-  Scenario(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
+  (title: string, callback: ICodeceptCallback): ScenarioConfig;
+  (title: string, opts: { [key: string]: any }, callback: ICodeceptCallback): ScenarioConfig;
 }
-declare function xScenario(title: string, callback: ICodeceptCallback): ScenarioConfig;
-declare function xScenario(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-declare interface IData {
-  Scenario(title: string, callback: ICodeceptCallback): ScenarioConfig;
-  Scenario(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-  only: IScenario;
+
+declare interface Scenario extends IScenario {
+  only: IScenario
+  skip: IScenario
 }
-declare function Data(data: any): IData;
-declare function xData(data: any): IData;
-declare function Before(callback: ICodeceptCallback): void;
-declare function BeforeSuite(callback: ICodeceptCallback): void;
-declare function After(callback: ICodeceptCallback): void;
-declare function AfterSuite(callback: ICodeceptCallback): void;
-
-declare function inject(): { [key: string]: any };
-declare function locate(selector: LocatorOrString): Locator;
-declare function within(selector: LocatorOrString, callback: Function): Promise<any>;
-declare function session(selector: LocatorOrString, callback: Function): Promise<any>;
-declare function session(selector: LocatorOrString, config: any, callback: Function): Promise<any>;
-declare function pause(): void;
-declare function secret(secret: any): string;
-
-declare const codeceptjs: any;
 
-declare namespace CodeceptJS {
-  export interface {{I}} {
-{{methods}}
-  }
-{{exportPageObjects}}
+interface IData {
+  Scenario: IScenario
+  only: {
+    Scenario: IScenario
+  };
 }
 
-declare module "codeceptjs" {
```

Comment:
Removed it because the implementation didn't match index.js,  was it used somewhere?